odml/doc.py

```diff
@@ -14,23 +16,29 @@ class Document(base._baseobj):
 class BaseDocument(base.sectionable, Document):
     """
     A represenation of an odML document in memory.
-
     Its odml attributes are: *author*, *date*, *version* and *repository*.
-
     A Document behaves very much like a section, except that it cannot hold
     properties.
     """
 
     _format = format.Document
 
-    def __init__(self, author=None, date=None, version=None, repository=None):
+    def __init__(self, author=None, date=None, version=None, repository=None, id=None):
         super(BaseDocument, self).__init__()
```

Comment:
Actually, yes, I think I should remove them from all constructors, since we only passing ids through parsers. I added them to constructor when I was using setters, so now they do not make sense.